compose/bin/create-user

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# Function to create admin user
+create_admin_user() {
```

Comment:
It doesn't make sense to me to create a generic, pre-filled admin account here. Let's always ask the user for this data instead.